src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy

```diff
@@ -89,6 +89,9 @@ class PalantirDockerPlugin implements Plugin<Project> {
                     buildCommandLine.addAll('--build-arg', "${buildArg.getKey()}=${buildArg.getValue()}")
                 }
             }
+            if (ext.pull) {
```

Comment:
That would be ideal indeed, but if we follow this logic arguments like `dockerfile` or `buildArgs` can be removed since it's possible to pass them as a map argument. My understanding of Gradle plugins is that they replace scripts and complex command lines by providing user friendly interfaces for common actions. I believe 'pull' is a very common parameter used on CI systems.